### ISSUE TITLE
Kitty Icon in OSX Menubar

### ISSUE THREAD

**Issue by jfcaron3** [Jermolene/TiddlyDesktop#117]
Is it possible to remove the Kitty icon in the OSX Menubar?  I can't seem to find the option that does this.

**Comment by Jermolene** [comment]
Hi @jfcaron3 I'm afraid that currently there's no option to remove the icon from the menu bar (I assume you can't remove it in the usual way by command-dragging it away from the menu bar?).

**Comment by taylorcoreyd** [comment]
Came here trying to figure out a way to remove the menubar tool as well.

I will confirm that command-dragging it away from the menu bar does not work.

**Comment by russellmcc** [comment]
Just wanted to chime in that not providing an option goes against apple's guidelines for menu bar extras: https://developer.apple.com/design/human-interface-guidelines/macos/extensions/menu-bar-extras/ .  How hard would it be to add an option for this?  I'm happy to try to do it if you could provide a pointer or two.

**Comment by dented42** [comment]
Given that the menubar item doesn't actually provide any functionality that clicking on the dock icon doesn't accomplish I would recommend in the strongest possible terms removing it altogether. Having a menubar item that does nothing that is impossible to remove feels actively user hostile.

**Comment by Jermolene** [comment]
Hi @dented42 on macOS the menulet currently provides direct access to the WikiList, Settings and Help windows of TiddlyDesktop. I do appreciate that clicking on the dock icon does almost the same thing by bringing all of the TiddlyDesktop windows to the front. I'd prefer to approach it by providing a config switch to disable the menulet for users who don't like it, but to make it more useful for those that do. Notably, I'd like to add a list of the open wiki windows.

**Comment by dented42** [comment]
I agree it's not a permanent solution, I was suggesting it more as a stopgap measure until it can be improved upon.

**Comment by Jermolene** [comment]
Hi @dented42 adding a new tab to settings with a checkbox to control visibility of the menulet wouldn't be much more complicated, if you might be interested in working on that? I'd be happy to help.

**Comment by dented42** [comment]
Sorry for the delay, yes that's definitely something I'm going to work on. I think I should be able to figure it out myself but I would appreciate someone looking over my patch to check that it's sane (my javascript is more than slightly rusty).
